docs/api.md

```diff
@@ -53,18 +53,18 @@ ReactDOM.render(
 
 Connects a React component to a Redux store.
 
-It does not modify the component class passed to it.  
```

Comment:
Spaces are significant in markdown.
Two spaces before the line break make newlines act like newlines.

Can you please remove all spacing changes?